pkgs/development/compilers/ghc/6.10.2-binary.nix

```diff
@@ -1,54 +1,62 @@
-{stdenv, lib, fetchurl, perl, libedit, ncurses5, gmp}:
+{ stdenv
+, fetchurl, perl
+, libedit, ncurses5, gmp
+}:
```

Comment:
I don't believe that a consistent style is important for those files and I would prefer to avoid any changes whatsoever in the old compiler builds unless there is a technical reason to make the change. Reformatting whitespace in the argument lists feels pointless to me.